eng/common/templates/steps/source-build.yml

```diff
@@ -23,7 +23,7 @@ steps:
     # In addition, add an msbuild argument to copy the WIP from the repo to the target build location.
     # This is because SetupNuGetSources.sh will alter the current NuGet.config file, and we need to preserve those
     # changes.
-    $internalRestoreArgs=
+    internalRestoreArgs=
```

Comment:
Change under `eng/common` would need an update in arcade repo (otherwise it will be overwritten by the next arcade update).

Reply:
ah thanks for catching that, didn't notice this was in eng/common for some reason 🤦

Reply:
moved it to https://github.com/dotnet/arcade/pull/8357